pkg/netdevice/pciNetDevice_test.go

```diff
@@ -247,13 +165,12 @@ var _ = Describe("PciNetDevice", func() {
 					"sys/bus/pci/devices/0000:00:00.1",
 					"sys/kernel/iommu_groups/0",
 					"sys/bus/pci/drivers/vfio-pci",
-					"dev/vhost-net",
+					// "dev/vhost-net", FIXME: /dev folder is not mocked
```

Comment:
why this change?

Reply:
To emphasize that this path is useless, since it is not mocked.

Reply:
so the path was useless before the refactoring ?

Reply:
Yes. For the testing purposes it make sense to implement it in the future